finagle/buoyant/src/main/scala/com/twitter/finagle/buoyant/TlsClientConfig.scala

```diff
@@ -6,7 +6,8 @@ import com.twitter.finagle.ssl.{KeyCredentials, TrustCredentials}
 import com.twitter.finagle.ssl.client.{SslClientConfiguration, SslClientEngineFactory}
 import com.twitter.finagle.transport.Transport
 import com.twitter.io.StreamIO
-import java.io.{File, FileInputStream, FileOutputStream}
+import java.io._
+
```

Comment:
I think IDEA adds those automatically - I'll have to find the config to make it not do that.